lldb/source/API/SBProcess.cpp

```diff
@@ -1244,6 +1244,17 @@ lldb::SBProcessInfo SBProcess::GetProcessInfo() {
   return sb_proc_info;
 }
 
+lldb::SBFileSpec SBProcess::GetCoreFile() {
+  LLDB_INSTRUMENT_VA(this);
+
+  ProcessSP process_sp(GetSP());
+  FileSpec core_file;
+  if (process_sp) {
+    core_file = process_sp->GetCoreFile();
+  }
+  return SBFileSpec(core_file);
+}
+
 lldb::addr_t SBProcess::AllocateMemory(size_t size, uint32_t permissions,
```

Comment:
move to SBTarget for consistency at the public API layer since we load the core file in SBTarget. If others believe that this should stay in SBProcess, let me know as I can go with what makes most sense, but the target seems the right location as I am thinking about it right now.

Reply:
We pass the filepath to `SBTarget::LoadCore` but I don't think this is a property of the Target, it seems like a property of the Process to me.  I suppose someone could create a Process with a core file, then close that Process, then create a new Process with a different core file, all in the same Target if the ArchSpecs were the same.